code/modules/atmospherics/FEA_fire.dm

```diff
@@ -15,19 +14,15 @@
 /turf/proc/hotspot_expose(exposed_temperature, exposed_volume, source_of_heat, electric = FALSE)
 	SHOULD_CALL_PARENT(TRUE)
 	src.material_trigger_on_temp(exposed_temperature)
-	if (src.reagents)
-		src.reagents.temperature_reagents(exposed_temperature, exposed_volume, 350, 300, 1)
-	if(!ON_COOLDOWN(src, "hotspot_expose_to_atoms__1", 1 SECOND) || !ON_COOLDOWN(src, "hotspot_expose_to_atoms__2", 1 SECOND) || \
-		!ON_COOLDOWN(src, "hotspot_expose_to_atoms__3", 1 SECOND) || !ON_COOLDOWN(src, "hotspot_expose_to_atoms__4", 1 SECOND) || \
-		!ON_COOLDOWN(src, "hotspot_expose_to_atoms__5", 1 SECOND))
-		if (electric) //mbc : i'm putting electric zaps on here because eleczaps ALWAYS happen alongside hotspot expose and i dont want to loop all atoms twice
-			for (var/atom/movable/item as anything in src)
-				item.temperature_expose(null, exposed_temperature, exposed_volume)
-				if (item?.flags & FLUID_SUBMERGE)
-					item.electric_expose(electric)
-		else
-			for(var/atom/movable/item as anything in src)
-				item.temperature_expose(null, exposed_temperature, exposed_volume)
+	src.reagents?.temperature_reagents(exposed_temperature, exposed_volume, 350, 300, 1)
+	if (electric) //mbc : i'm putting electric zaps on here because eleczaps ALWAYS happen alongside hotspot expose and i dont want to loop all atoms twice
```

Comment:
no cooldown?

Reply:
yass said it was prooobably safe to remove them since it was targeted at giant hotboxes with hundreds of weed, which has been made to not happen anymore